Core/detection.js

```diff
@@ -33,8 +33,12 @@
             type = event.target.nodeName
         }
 
+        duckduckgoMessaging.log("checking " + event.url + " (" + type + ")");
         duckduckgoContentBlocking.shouldBlock(event.url, type, function(url, block) {
-            if (!block) { return }
+            if (!block) { 
+                duckduckgoMessaging.log("don't block " + url);
+                return 
+            }
```

Comment:
I admit I may be too careful here (as I don't know the exact cost) but wouldn't these calls add extra overhead? I'm all for logging these in debug (development) but I'm unsure about pushing too many events from JS to WebKit on production (especially, if they are going to be silenced anyway by Logger's `#if DEBUG`).